src/pages/GivePage/EpochStatementDrawer.tsx

```diff
@@ -76,7 +76,7 @@ export const EpochStatementDrawer = ({
 
   const { mutate: updateEpochStatement } = useMutation(
     async (bio: string) => updateUser({ circle_id: member.circle_id, bio }),
-    { onSuccess: () => setSaving('saved') }
+    { onSettled: (data, error) => setSaving(error ? 'error' : 'saved') }
```

Comment:
`data` variable is never used, is there a way to ignore it? I wasn't finding a way, or maybe rename it to `_` ?

Reply:
you just do onSettled: (,error) =>

Reply:
nevermind that only works with tuples, i've read others thoughts on this and it seems you just leave it in there, and it isn't an error of unused param because its "use" is taking up that slot in the argument list